### PR TITLE
Remove data from repo

### PR DESCRIPTION
**Pull request by sbalandi**
In the below notebooks it is changed getting data from repo to downloading from storage, changed getting notebook_utils from repo to loading from github and added link to Colab:

Tieckts: _CVS-115590_

- [001-hello-world](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/001-hello-world/001-hello-world.ipynb) 
- [003-hello-segmentation](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/003-hello-segmentation/003-hello-segmentation.ipynb) 
- [004-hello-detection](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/004-hello-detection/004-hello-detection.ipynb) 
- [101-tensorflow-classification-to-openvino](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/101-tensorflow-classification-to-openvino/101-tensorflow-classification-to-openvino.ipynb) 
- 102 
     -  [102-pytorch-to-openvino](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/102-pytorch-to-openvino/102-pytorch-to-openvino.ipynb) 
     - [102-pytorch-onnx-to-openvino](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/102-pytorch-to-openvino/102-pytorch-onnx-to-openvino.ipynb)
- [103-paddle-to-openvino-classification](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb)
- [104-model-tools](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/104-model-tools/104-model-tools.ipynb) 
- [105-language-quantize-bert](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb) 
- [106-auto-device](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/106-auto-device/106-auto-device.ipynb) 
- 109 
     - [109-latency-tricks](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/109-performance-tricks/109-latency-tricks.ipynb) 
     - [109-throughput-tricks ](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/109-performance-tricks/109-throughput-tricks.ipynb)
- [113-image-classification-quantization ](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/113-image-classification-quantization/.ipynb)
- 117 (?)
- [118-optimize-preprocessing](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb)
- [207-vision-paddlegan-superresolution](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/207-vision-paddlegan-superresolution/207-vision-paddlegan-superresolution.ipynb)
- [208-optical-character-recognition](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/208-optical-character-recognition/208-optical-character-recognition.ipynb)
- [209-handwritten-ocr](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/209-handwritten-ocr/209-handwritten-ocr.ipynb) 
- [211-speech-to-text ](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/211-speech-to-text/211-speech-to-text.ipynb)
- [216-attention-center](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/216-attention-center/216-attention-center.ipynb) 
- [219-knowledge-graphs-conve.](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/219-knowledge-graphs-conve/219-knowledge-graphs-conve.ipynb)
- [224-3D-segmentation-point-clouds](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/224-3D-segmentation-point-clouds/224-3D-segmentation-point-clouds.ipynb)
- [225-stable-diffusion-text-to-image](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb)
- [229-distilbert-sequence-classification](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/229-distilbert-sequence-classification/229-distilbert-sequence-classification.ipynb)
- [403-action-recognition-webcam](https://colab.research.google.com/github/sbalandi/openvino_notebooks/blob/data_move/notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb)

### CODE REVIEW

**Comment by eaidova** [comment]
@sbalandi looks like something wrong with 207 notebook 
ModuleNotFoundError: No module named 'google.colab'

**Review by eaidova — APPROVED** [review]

File: notebooks/001-hello-world/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Introduction to OpenVINO™
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F001-hello-world%2F001-hello-world.ipynb)
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/001-hello-world/001-hello-world.ipynb)
```
Comment (igor-davidyuk) [inline notebooks/001-hello-world/README.md:3]: Can't we have both binder and colab?

Comment (sbalandi) [reply notebooks/001-hello-world/README.md:4]: binders are restored

File: notebooks/229-distilbert-sequence-classification/README.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 Sequence Classification (or Text Classification) is the NLP task of predicting a label for a sequence of words.
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F229-distilbert-sequence-classification%2F229-distilbert-sequence-classification.ipynb)
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/229-distilbert-sequence-classification/229-distilbert-sequence-classification.ipynb)
```
Comment (igor-davidyuk) [inline notebooks/229-distilbert-sequence-classification/README.md:5]: It does not work in Colab

ModuleNotFoundError: No module named 'transformers'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.

**Review by igor-davidyuk — APPROVED** [review]
Looks good

**Pull request merged**